src/operators.jl

```diff
@@ -385,14 +444,18 @@ ops_2_to_1 = union(
 # Operators for functions f: ℝ → ℝ² #
 #===================================#
 
+function is_influence_out1_zero_global end
+function is_influence_out2_zero_global end
 function is_firstder_out1_zero_global end
 function is_seconder_out1_zero_global end
 function is_firstder_out2_zero_global end
 function is_seconder_out2_zero_global end
 
 # Fallbacks for local derivatives:
-is_seconder_out1_zero_local(f::F, x) where {F} = is_seconder_out1_zero_global(f)
+is_influence_out1_zero_local(f::F, x) where {F} = is_influence_out1_zero_global(f)
+is_influence_out2_zero_local(f::F, x) where {F} = is_influence_out2_zero_global(f)
 is_firstder_out1_zero_local(f::F, x) where {F} = is_firstder_out1_zero_global(f)
+is_seconder_out1_zero_local(f::F, x) where {F} = is_seconder_out1_zero_global(f)
```

Comment:
this was missing

Reply:
What do you mean?

Reply:
take a look at the source, `is_secondder_out1_zero_local` was never defined

Reply:
<img width="806" alt="image" src="https://github.com/adrhill/SparseConnectivityTracer.jl/assets/20258504/510ba37b-fa6e-4cc5-ac38-eedfaa5197b8">

Reply:
oh my bad it wasn't missing, just in a surprising order, I remember now